roles/network-multus/tasks/deprovision.yml

```diff
@@ -25,6 +25,40 @@
 - name: Delete cni plugins Resources
   command: "{{ cluster_command }} delete -f /tmp/cni-plugins.yml --ignore-not-found"
 
+# note: this rule does NOT delete /etc/pcidp/config.json file because k8s
+# doesn't support finalizers to clean up mounted directory
+- name: Uninstall SR-IOV device plugin
+  block:
+  - name: Render SR-IOV DP deployment yaml
+    template:
+      src: sriovdp.yml
+      dest: /tmp/sriovdp.yml
```

Comment:
should this only be run when sriov is enabled?

Reply:
+1  @booxter we can wrap everything in a block. and check for the sriov boolean